doozerlib/cli/scan_sources.py

```diff
@@ -265,30 +265,30 @@ def _detect_rhcos_status(runtime, kubeconfig) -> list:
     for arch in runtime.arches:
         for private in (False, True):
             name = f"{version}-{arch}{'-priv' if private else ''}"
-            tagged_mosc_id = _tagged_mosc_id(kubeconfig, version, arch, private)
+            tagged_rhcos_id = _tagged_rhcos_id(kubeconfig, rhcos.get_primary_container_name(runtime), version, arch, private)
```

Comment:
Do we need to check other tags? Are they guaranteed to be updated at the same time?